Fresh install fails

Fresh installation of the project fails with dependency resolution errors preventing the development server from starting.

## Environment
- Node.js: v21.7.3
- npm: 10.5.0
- OS: macOS

## Reproduction Steps

1. Clone the repository
2. Install dependencies:
   ```bash
   npm install
   ```
3. Start the development server:
   ```bash
   npm run dev
   ```

## Expected Behavior
The development server should start successfully at `http://localhost:5173`

## Actual Behavior
The installation and server startup fail with multiple errors:

### Error 1: Missing `react-is` dependency
```
✘ [ERROR] Could not resolve "react-is"

    node_modules/@carbon/ibm-products/es/node_modules/prop-types/index.js:9:23:
      9 │ import require$$0 from 'react-is';
        ╵                        ~~~~~~~~~~

Error: Error during dependency optimization
```

**Workaround:** Install `react-is` manually:
```bash
npm install react-is
```

### Error 2: Missing export from `@carbon-labs/utilities`
After installing `react-is`, the server starts but fails with:

```
SyntaxError: The requested module '../node_modules/@carbon-labs/utilities/es/usePrefix.js' does not provide an export named 'usePrefix'
    at file:///Users/alexseymour/datalabs/templates/carbon-react-router-starter/node_modules/@carbon-labs/react-theme-settings/es/components/ThemeSettings.js:12
```

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
     "express": "^5.0.1",
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
+    "react-is": "^19.0.0",
     "react-router": "^7.5.2",
     "sirv": "^3.0.0"
   },

